tests/transforms/test_broadcast_expand.py

```diff
@@ -107,33 +94,23 @@ def test_expansion(self, params, size, obs, exp_fn):
         result = fn(qml.execute(tapes, dev, None))
         expected = exp_fn(*params)
 
-        if len(tape.measurements) > 1 and size == 1:
-            expected = expected.T
-
         assert qml.math.allclose(result, expected)
 
-    @pytest.mark.parametrize("params, size", parameters_and_size)
+    @pytest.mark.parametrize("params", parameters)
     @pytest.mark.parametrize("obs, exp_fn", observables_and_exp_fns)
-    def test_expansion_qnode(self, params, size, obs, exp_fn):
+    def test_expansion_qnode(self, params, obs, exp_fn):
         """Test that the transform integrates correctly with the transform program"""
 
         @qml.transforms.broadcast_expand
         @qml.qnode(dev)
         def circuit(x, y, z, obs):
             qml.StatePrep(np.array([1, 0, 0, 0]), wires=[0, 1])
-            RX_broadcasted(x, wires=0)
-            qml.PauliY(0)
-            RX_broadcasted(y, wires=1)
-            RZ_broadcasted(z, wires=1)
-            qml.Hadamard(1)
+            _ = make_ops(x, y, z)
             return [qml.expval(ob) for ob in obs]
 
         result = circuit(*params, obs)
         expected = exp_fn(*params)
 
-        if len(obs) > 1 and size == 1:
-            expected = expected.T
```

Comment:
The extraction of the first entry of `out` in the `exp_fn`s takes care of this already. There is nothing to transpose because in this case there only is one axis in `expected`.